cli/flox-rust-sdk/src/providers/services.rs

```diff
@@ -261,7 +261,7 @@ impl ProcessStates {
 
 /// Constructs a base `process-compose process` command to which additional
 /// arguments can be appended.
-fn base_process_compose_command(socket: impl AsRef<Path>) -> Command {
+pub fn base_process_compose_command(socket: impl AsRef<Path>) -> Command {
```

Comment:
I cant find references to this outside of this module

```suggestion
fn base_process_compose_command(socket: impl AsRef<Path>) -> Command {
```

Reply:
Fixed